ops/deployer/gauntlet/gauntlet.go

```diff
@@ -0,0 +1,534 @@
+		t.gauntlet.Flag("network", t.network),
+	)
+	if err != nil {
+		return errors.New("Request AC initialization failed")
+	}
+	report, err := t.gauntlet.ReadCommandReport()
+	if err != nil {
+		return err
+	}
+	t.Account[RequesterAccessController] = report.Responses[0].Contract
+
+	fmt.Println("Step 2: Init Billing Access Controller")
+	err = t.gauntlet.ExecCommand(
+		"access_controller:deploy",
+		t.gauntlet.Flag("network", t.network),
+	)
+	if err != nil {
+		return errors.New("Billing AC initialization failed")
+	}
+	report, err = t.gauntlet.ReadCommandReport()
+	if err != nil {
+		return err
+	}
+	t.Account[BillingAccessController] = report.Responses[0].Contract
+
+	fmt.Println("Step 6: Init OCR 2 Feed")
+	input := OCRinit{
+		MinAnswer:                 "0",
+		MaxAnswer:                 "10000000000",
+		Decimals:                  2,
+		Description:               "Hello",
+		RequesterAccessController: t.Account[RequesterAccessController],
+		BillingAccessController:   t.Account[BillingAccessController],
+		LinkToken:                 t.Account[LINK],
+	}
+
+	jsonInput, err := json.Marshal(input)
+	if err != nil {
+		return err
+	}
+
+	err = t.gauntlet.ExecCommand(
+		"ocr2:deploy",
+		t.gauntlet.Flag("network", t.network),
+		t.gauntlet.Flag("input", string(jsonInput)),
+	)
+	if err != nil {
+		return errors.New("feed initialization failed")
+	}
+
+	report, err = t.gauntlet.ReadCommandReport()
+	if err != nil {
+		return err
+	}
+
+	t.Account[OCR2] = report.Responses[0].Contract
+	fmt.Printf(" - %s", report.Data["state"])
+	return msg.Check(nil)
+}
+
+func (t GauntlerDeployer) TransferLINK() error {
+	msg := utils.LogStatus("Sending LINK to OCR contract")
+
+	err := t.gauntlet.ExecCommand(
+		"token:transfer",
+		t.gauntlet.Flag("network", t.network),
+		t.gauntlet.Flag("to", t.Account[OCR2]),
+		t.gauntlet.Flag("amount", "1000000000"),
+		t.gauntlet.Flag("link", t.Account[LINK]),
+		t.Account[LINK],
+	)
+	if err != nil {
+		return errors.New("LINK transfer failed")
+	}
+
+	return msg.Check(err)
+}
+
+const BeginProposal = "begin_proposal"
+
+type ProposeConfigDetails struct {
+	ID            string   `json:"proposalId"`
+	Payees        []string `json:"payees"`
+	Signers       []string `json:"signers"`
+	Transmitters  []string `json:"transmitters"`
+	F             uint8    `json:"f"`
+	OnchainConfig []byte   `json:"onchainConfig"`
+}
+
+type ProposeOffchainConfigDetails struct {
+	ID                    string                `json:"proposalId"`
+	OffchainConfigVersion uint64                `json:"offchainConfigVersion"`
+	OffchainConfig        OffchainConfigDetails `json:"offchainConfig"`
+}
+
+type ReportingPluginConfig struct {
+	AlphaReportInfinite bool          `json:"alphaReportInfinite"`
+	AlphaReportPpb      uint64        `json:"alphaReportPpb"`
+	AlphaAcceptInfinite bool          `json:"alphaAcceptInfinite"`
+	AlphaAcceptPpb      uint64        `json:"alphaAcceptPpb"`
+	DeltaCNanoseconds   time.Duration `json:"deltaCNanoseconds"`
+}
```

Comment:
Search the project for these types. They are now duplicated and used across `terrad`, `gauntlet` & `e2e` deployers. Please do some cleanup and consolidate these types in one place.